menu.yml

```diff
@@ -12,5 +14,5 @@ scenarios:
   - flow:
     - loop:
       - get:
-          url: "/restaurants/voluptatem9999975/menu/lunch/vegan"
-      count: 1000
+          url: "{{uri}}"
```

Comment:
Wouldn't this configuration loop over each url in your csv file 300 times? I think you might need an 'over:' though i'm not sure about this. Let me know if i'm wrong! Putting in a ```"log: 'this is the url: {{uri}}"``` will help debug if you haven't tried that.